addons/print_sale/wizard/print_order_wizard.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from openerp import api, models
+
+class print_order_line_wizard(models.TransientModel):
+
+    _inherit = 'print.order.line.wizard'
+
+    @api.model
+    def _default_selection_state(self):
+        selection = super(print_order_line_wizard, self)._default_selection_state()
+        selection.append(('bad_sale_order_state', 'Wrong Sale Order State'))
```

Comment:
Keys are weird ... probably they should be prefixed by the model, then the state: sale_order_wrong_state for example. So they will be easier to understand.